### DISCUSSION TITLE
PUAN SIRANIZI ÖĞRENİN

### DISCUSSION THREAD

**Discussion by Omertastan2036** [IBAX-io/go-ibax#1327] [Q&A]
Sağdaki üç çizgiye tıklayın DEVELOPMENT'e tıklayın 
Honor Developents'e tıklayıp puan sıralamasını öğrenebilirsiniz.

**Comment by Melike6606** [comment]
Çok teşekkür ederim sağol

**Reply by Omertastan2036** [reply]
Rica ederim

**Reply by Skips1** [reply]
Hangi sağdaki 3 çizgi kanka göremedim ?

**Reply by Skips1** [reply]
Tamam gördüm kardeşim sağol

**Reply by Omertastan2036** [reply]
Ne demek

**Comment by sadikyigit** [comment]
Sağolun hocam

**Reply by Omertastan2036** [reply]
Rica ederim

**Comment by Fatihkanlii** [comment]
Hocam sizlere minnettarız.

**Reply by Omertastan2036** [reply]
Eyvallah

**Comment by Mustafasssn** [comment]
Bunu bilmiyordum tesekkurler
Projeye inancim gitmisti ilk 1000 de olsugumu farkettim tesekkurler hocam

**Reply by Omertastan2036** [reply]
İlk 1000 e 500 ibx verecek ben de dedim olmaz ama ilk 1000 deyim sanırım

**Comment by Sevdeozgok** [comment]
Teşekkürler hocam çok sağolun çok ıyısınız

**Reply by Omertastan2036** [reply]
Rica ederim

**Comment by Aysenuraysenuraysenur** [comment]
Projeye olan inancım bitmişti ilk 1000 de olduğumu farkettim teşekkürler hocam

**Reply by Omertastan2036** [reply]
Rica ederim

**Comment by Omertastan2036** [comment]
İlk 1000 e 500ibx verecek arkadaşlar ona göre

**Reply by Erbil54** [reply]
sıralamayı nerden görebilirim

**Reply by Omertastan2036** [reply]
Anlattım ya knk siran yazmıyor ama puanına göre kendini bul yukariya doğru sayarak git

**Reply by Omertastan2036** [reply]
Anlattım ya knk siran yazmıyor ama puanına göre kendini bul yukariya doğru sayarak git

**Comment by oolukman** [comment]
Senior scout ta 348, infantry da 4447. görünüyorum. Bunlar ne anlama geliyor bilen bir arkadaş açıklayabilir mi?

**Reply by Omertastan2036** [reply]
O senin sıralaman değil 
Grup grup ayrılmış sen o 4k kişi içindesin ama 4 bininci değilsin 
Sayılar herkeste aynı puanını söyle tahmini sıralamanı soyleyeyim

**Reply by Omertastan2036** [reply]
Hayır o puan sıralamasında kimin kaçıncı sırada olduğu yazsa daha iyi olur @ibaxio

**Reply by Omertastan2036** [reply]
Yoksa tek tek saymak zorundayız

**Reply by oolukman** [reply]
Anlamadım. Puan 22736, ref 18 coin

**Reply by eraydgann** [reply]
Thanks.

**Reply by Omertastan2036** [reply]
26 dan aşağısı 1k nin altinda kalir

**Reply by eraydgann** [reply]
Thx Ömer .

**Comment by eraydgann** [comment]
Sağ ol Omer

**Reply by Omertastan2036** [reply]
Rica ederim
